tests/multinode.rs

```diff
@@ -1734,9 +1735,14 @@ fn test_broadcast_last_tick() {
     let blob_fetch_stages: Vec<_> = listening_nodes
         .iter_mut()
         .map(|(_, _, node, _)| {
-            BlobFetchStage::new(
-                Arc::new(node.sockets.tvu.pop().unwrap()),
-                blob_receiver_exit.clone(),
+            let (blob_fetch_sender, blob_fetch_receiver) = channel();
```

Comment:
Can this test be in-ignored?  Seems like it’s the only test that would be affected by this patch, which really means that it’s all untested (beyond running a full-blown testnet)

Reply:
There are tests in TVU which should exercise this change, as BlobFetchStage is part of TVU.
I am not sure why this test is ignored. Maybe a separate PR to test/unignore it?